### ISSUE TITLE
Display input arguments to fortls in debug logs

### ISSUE THREAD

**Issue by gnikit** [fortran-lang/fortls#348]
Display a serialised version of the "final" (config file + CLI) options passed to fortls in the debug logs.

**Comment by AnonMiraj** [comment]
just to make sure 
does this mean it should list all the options in the debug log whenever fortls starts ?
if that is so i could work on it.

**Comment by gnikit** [comment]
Yes, these would be the combined options of the CLI + configuration file. Feel free to open a PR.
